examples/quickstarts/helidon-quickstart-se/src/test/java/io/helidon/examples/quickstart/se/MainTest.java

```diff
@@ -17,90 +17,95 @@
 package io.helidon.examples.quickstart.se;
 
 import java.util.Collections;
-import java.util.concurrent.TimeUnit;
 
 import javax.json.Json;
 import javax.json.JsonBuilderFactory;
 import javax.json.JsonObject;
 
-import io.helidon.media.jsonp.JsonpSupport;
+import io.helidon.common.http.Http;
 import io.helidon.webclient.WebClient;
 import io.helidon.webclient.WebClientResponse;
 import io.helidon.webserver.WebServer;
+import io.helidon.webserver.testsupport.SetUpServer;
+import io.helidon.webserver.testsupport.WebServerTest;
 
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+@WebServerTest
 class MainTest {
 
-    private static WebServer webServer;
-    private static WebClient webClient;
     private static final JsonBuilderFactory JSON_BUILDER = Json.createBuilderFactory(Collections.emptyMap());
-    private static final JsonObject TEST_JSON_OBJECT;
 
-    static {
-        TEST_JSON_OBJECT = JSON_BUILDER.createObjectBuilder()
-                .add("greeting", "Hola")
-                .build();
-    }
-
-    @BeforeAll
-    static void startTheServer() {
-        webServer = Main.startServer().await();
+    private final WebClient webClient;
 
-        webClient = WebClient.builder()
-                .baseUri("http://localhost:" + webServer.port())
-                .addMediaSupport(JsonpSupport.create())
-                .build();
+    MainTest(WebClient webClient) {
+        this.webClient = webClient;
```

Comment:
This looks weird to me without the `@Inject`, maybe injecting webclient to each test method as parameter would look better?